.travis.yml

```diff
@@ -77,3 +77,7 @@ script:
 
 notifications:
   email: false
+
+branches:
+  exclude:
+    - /^sils\//
```

Comment:
the newline is there @Techievena  you can see it's missing by github showing a red icon at the end of the line